docs/BuyButton.md

```diff
@@ -2,21 +2,27 @@
 
 # Buy Button
 
-The `BuyButton` block allows users to **add products in the minicart**.
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+The `buy-button` block allows users to add products in the [Minicart](https://vtex.io/docs/components/all/vtex.minicart@2.46.1/) (`minicart.v1`).
+
+:warning: **The Buy Button block only effectively function i.e. only adds products to the Minicart if the store uses the Minicart v1**. When using the Minicart v2, the [Add To Cart Button](https://vtex.io/docs/components/all/vtex.add-to-cart-button@0.9.0/) should be used instead.
 
 ![image](https://user-images.githubusercontent.com/284515/70233985-69e13700-173e-11ea-91f7-6675a6a0e73b.png)
 
 ## Configuration
 
 1. Import the `vtex.store-component` app to your theme's dependencies in the `manifest.json`;
 
-```json
+```diff
   "dependencies": {
-    "vtex.store-components": "3.x"
++   "vtex.store-components": "3.x"
   }
 ```
 
-2. Add the `buy-button` block to any block bellow `store.product` (Product template). For example:
+2. Add the `buy-button` block to other blocks from your theme according to your needs. In the example below, the block is added to the `flex-layout.row` block from the `store.product` template:
```

Comment:
This passes the idea that it will work in _any_ block, but I believe it really needs to be bellow `store.product` for it to work!